packages/teleport-test/src/standalone.ts

```diff
@@ -49,6 +49,11 @@ const run = async () => {
     console.info(ProjectType.PREACT, '-', result.payload)
     result = await packProject(projectUIDL, { ...packerOptions, projectType: ProjectType.ANGULAR })
     console.info(ProjectType.ANGULAR, '-', result.payload)
+    result = await packProject(projectUIDL, {
```

Comment:
can the packProject function decide which template to use internally? based on the publisher